.travis.yml

```diff
@@ -34,17 +43,14 @@ install:
   - drush --uri=127.0.0.1:8282 en -y islandora_audio islandora_breadcrumbs islandora_iiif islandora_image islandora_video islandora_text_extraction_defaults
   - drush --uri=127.0.0.1:8282 fim -y islandora_core_feature,islandora_text_extraction_defaults
 
+before_script:
+  - mkdir -p $DRUPAL_DIR/web/sites/simpletest/browser_output
+  - cp $PHPUNIT_FILE $DRUPAL_DIR/web/core
+
 script:
   - $SCRIPT_DIR/travis_scripts.sh
-  - $SCRIPT_DIR/run-tests.sh "islandora"
-  - $SCRIPT_DIR/run-tests.sh "islandora_breadcrumbs"
-  - $SCRIPT_DIR/run-tests.sh "islandora_image"
-  - $SCRIPT_DIR/run-tests.sh "islandora_audio"
-  - $SCRIPT_DIR/run-tests.sh "islandora_video"
-  - $SCRIPT_DIR/run-tests.sh "islandora_text_extraction"
-
-after_success:
-  - bash <(curl -s https://codecov.io/bash)
```

Comment:
If you add this line back in at the end of `after_success` will we get codecov back?  I'm trying to think about how it would all play out in this scenario.  Is it possible to still have codecov with this approach?

Reply:
We don't use codecov for Drupal modules do we?

Reply:
You can have codecov, I disabled xdebug to get the speed increase and because we didn't seem to ever use codecov for Drupal modules. It can be re-enabled if you desire.